woocommerce/woocommerce-android
# [Hazmat Shipping] Add hazmat tracks

Summary
==========
Introduce a set of Tracks to the Shipping Label creation flow regarding Hazmat package declarations. The following tracks were added:

- CONTAINS_HAZMAT_CHECKED -> When the `Contains Hazardous Materials` toggle is activated
- HAZMAT_CATEGORY_SELECTOR_OPENED -> When the Hazmat category selector is displayed to the user
- HAZMAT_CATEGORY_SELECTED -> When the user selects an Hazmat category
   **Parameters**: 
   - Category: The selected category
   - Order ID: ID of the order with hazmat content under the Shipping Label creation process

How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow with valid US address options until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Hit the `Contains Hazardous Materials` toggle and verify that `🔵 Tracked: contains_hazmat_checked` is triggered.
5. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration
6. Hit the `Select a category` button and make sure the `🔵 Tracked: hazmat_category_selector_opened` is triggered.
7. Select a hazmat option and make sure the `🔵 Tracked: hazmat_category_selected` is triggered.

Update release notes:
